api/envoy/api/v2/route/route.proto

```diff
@@ -132,6 +132,9 @@ message VirtualHost {
   // :ref:`suppress_envoy_headers
   // <envoy_api_field_config.filter.http.router.v2.Router.suppress_envoy_headers>` flag.
   bool include_request_attempt_count = 14;
+
+  // Indicates the retry policy for all routes in this virtual host.
```

Comment:
Would be good to document the interaction this has with a per route retry policy, specifically which one has precedence